lib/features/navigation_tab_view/navigation_tab_view.dart

```diff
@@ -0,0 +1,109 @@
+import "package:auto_route/auto_route.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../config/ui_config.dart";
+import "../../theme/app_theme.dart";
+import "../../utils/context_extensions.dart";
+import "../analytics/show_feedback_tile.dart";
+import "../bottom_nav_bar/bottom_nav_bar_icon_icons.icons.dart";
+import "../home_view/widgets/logo_app_bar.dart";
+import "../navigator/utils/navigation_commands.dart";
+import "widgets/about_the_app_tile.dart";
+import "widgets/about_us_tile.dart";
+import "widgets/navigation_tile.dart";
+import "widgets/notification_button.dart";
+import "widgets/small_tile.dart";
+
+@RoutePage()
+class NavigationTabView extends ConsumerWidget {
+  const NavigationTabView({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final children = [
+      const SizedBox(height: NavigationTabViewConfig.universalPadding),
+      const AboutUsTile(),
+      const ShowFeedbackTile(),
+      const SizedBox(height: NavigationTabViewConfig.universalPadding / 3),
+      _NavigationRow(
+        child1: SmallTileCard(
+          onTap: ref.navigateDepartments,
+          title: context.localize.departments,
+          icon: const Icon(BottomNavBarIcon.faculty_icon, size: 30),
+        ),
+        child2: SmallTileCard(
+          onTap: () {
+            // TODO(simon-the-shark): implement review request
+          },
+          title: context.localize.leave_a_review,
+          icon: const Icon(Icons.star, color: Color(0xFFFFDF00), size: 30),
+        ),
+      ),
+      _NavigationRow(
+        child1: SmallTileCard(
+          onTap: ref.navigateToSksMenu,
+          title: context.localize.sks_full_name,
+          icon: const Icon(Icons.restaurant_menu, size: 30),
+        ),
+        child2: SmallTileCard(
+          onTap: ref.navigateScienceClubs,
+          title: context.localize.student_organizations,
+          icon: const Icon(BottomNavBarIcon.sci_circle_icon, size: 24),
+        ),
+      ),
+      Padding(
+        padding: const EdgeInsets.only(top: 16),
+        child: Text(
+          context.localize.rest_header,
+          style: context.textTheme.headline,
+        ),
+      ),
+      NavigationTile(
+        onTap: () {},
+        title: context.localize.settings,
+        icon: Icons.settings,
+      ),
+      const AboutTheAppTile(),
+      const SizedBox.shrink(),
```

Comment:
Why `SizedBox `here?

Reply:
This generally is a little hacky, but this gives us extra bottom padding as we're using List.separated later. Do you think I should change it to Padding wrapped around bottom widget instead?

Reply:
Yeah, I see, imo better to wrap padding around bottom widget - self explanatory